numerics/root_finders_body.hpp

```diff
@@ -69,10 +71,11 @@ Argument Brent(Function f,
   using Value = decltype(f(lower_bound));
   Value const zero{};
 
-  // We do not use std::numeric_limits<double>::epsilon, because it is 2ϵ in
-  // Brent’s notation: Brent uses ϵ = β^(1-τ) / 2 for rounded arithmetic, thus
-  // here ϵ = 2^-τ, see (2.9).
-  constexpr double ϵ = 0x1p-53;
+
+  // We do not use |std::numeric_limits<double>::epsilon()|, because it is 2ϵ in
+  // Brent’s notation: Brent uses ϵ = β^(1-τ) / 2 for rounded arithmetic, see
+  // (2.9).
+  constexpr double ϵ = ScaleB(0.5, 1 - std::numeric_limits<double>::digits);
```

Comment:
I don't understand how this formula relates to the one in the comment.  I would have expected `ScaleB(radix, 1 - digits) * 0.5`.  What am I missing?

Reply:
> What am I missing?

The definition of `scaleB` from IEEE 754-2008.
> **scaleB**(𝑥, 𝛮) is 𝑥×𝑏<sup>𝛮</sup> for integral values 𝛮.